app/src/main/java/edu/agh/susgame/front/ui/component/menu/CreateGameView.kt

```diff
@@ -1,32 +1,182 @@
 package edu.agh.susgame.front.ui.component.menu
 
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Visibility
+import androidx.compose.material.icons.filled.VisibilityOff
 import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Slider
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+
 import androidx.navigation.NavController
 import edu.agh.susgame.front.ui.Translation
 import edu.agh.susgame.front.ui.component.common.Header
 import edu.agh.susgame.front.ui.component.menu.navigation.MenuRoute
 import edu.agh.susgame.front.ui.theme.PaddingL
 
+import java.util.Locale
+import kotlin.math.roundToInt
+
+
+private const val maxPinLength = 6
+private const val maxPlayersAmount = 6
+private const val minPlayersAmount = 2
+private const val defaultPlayersAmount = 4
+private const val minGameTime = 5f
+private const val maxGameTime = 15f
+private const val defaultGameTime = 10
 @Composable
 fun CreateGameView(
```

Comment:
Jest tutaj taki problem, że gdy telefon jest poziomo, to nie wszystko się mieści, a nie da się przewinąć w dół

![image](https://github.com/Nepommuck/SusGameFrontend/assets/100998310/e73b9912-ecbe-45dd-98c9-52604f7cb569)

Obczaj, jak to jest zrobione w `AwaitingGameView`. Żeby faktycznie zabrakło miejsca, dodaj więcej elementów do `MockAwaitingGamesProvider#state`,

Reply:
naprawione, dobrze ze podpowiedziales gdzie tego poszukac, bardzo fajna opcja